avocado/utils/process.py

```diff
@@ -222,6 +222,7 @@ class CmdResult(object):
     :param stdout: String containing stdout of the process
     :param stderr: String containing stderr of the process
     :param duration: Elapsed wall clock time running the process
+    :param pid: ID of the process
```

Comment:
I'd add the attribute `pid` to `CmdResult`, set to `None` by default, and document it with a line such as:

```
#: Contains the process ID or None
self.pid = None
```

This should allow for the attribute to be shown in the docs.